nss-day-cohort-29/nutshell-midnight-dream
# Added HTML structure for news articles list

# Description
Added HTML structure for news articles list.

## Type of change

# Testing Instructions for Change Made
1. `git fetch --all`
2. `git checkout meg-news-articles-setup`
3. Run grunt
4. Verify that the DOM displays the news page, which includes the title, "Add New Article" button, and list of articles (currently only one).

# Checklist:
